### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by dcoa**
This PR tries to update the Distro documentation to make it more accurate to the actual behavior. 

[JIRA ISSUE DS-904](https://edunext.atlassian.net/browse/DS-904?atlOrigin=eyJpIjoiY2U5ODdhMDI1YzlmNDE5ZTkwYWEzZGZlMDZiNWNkZGUiLCJwIjoiaiJ9)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,13 +37,13 @@ tutor distro run-extra-commands
 
 Distro plugin manages a set of settings that you can configure, to know how to do that check:
 
-- [How to custumize distro](./docs/how_to_customize_distro.rst)
+- [How to customize distro](./docs/how_to_customize_distro.rst)
```
Comment (bra-i-am) [inline README.md:40]: I think will be necessary to modify https://github.com/eduNEXT/tutor-contrib-edunext-distro/blob/master/docs/how_to_customize_distro.rst to add the extra-commands `DISTRO_EXTRA_COMMANDS`

File: README.md [hunk 2]
```diff
@@ -2,16 +2,20 @@
 
 ## What is distro
```
Comment (MaferMazu) [inline README.md:3]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -2,16 +2,20 @@
...
 
-Distro is an opinioned openedx distribution with some custom stuff to have an easy-to-use
-and a ready to deploy in local or in development openedx distribution.
-This can be watch like a tutor-plugin but is taken a little bit far away.
+Distro is a tool to facilitate the customization of an Openedx instance, adding commands and settings to have an easy-to-use
```
Comment (MaferMazu) [inline README.md:5]:
```suggestion
This plugin is a tool to facilitate the customization of an Openedx instance, adding commands and settings to have an easy-to-use
```

**Review by MaferMazu — COMMENTED** [review]
These changes look good, @dcoa. I only have two suggestions to be more precise and avoid combining concepts: remove the "What is distro" subtitle and talk about this plugin as this plugin instead of distro.

File: README.md (hunk 1 above)
Comment (dcoa) [reply README.md:39]: done

File: README.md (hunk 3 above)
Comment (dcoa) [reply README.md]: done

**Review by bra-i-am — APPROVED** [review]

**Review by MaferMazu — APPROVED** [review]

**Pull request merged**